### PR TITLE
Remove dependency resolver, helium and references to them in configur…

### LINKED ISSUE #36: Remove the whole dependency resolver along with it's tests
please create an issue about removing the whole dependency resolver along with it's tests

_Originally posted by @kortemik in https://github.com/teragrep/zep_01-migrate/pull/24#discussion_r1842140671_
            

[zeppelin-interpreter/src/test/java/org/apache/zeppelin/dep/DependencyResolverTest.java](https://github.com/teragrep/zep_01-migrate/pull/24/files/a9356ba8c4126b793a54c95cd1b903f1042dea34#diff-ce8602dd82adfe1435df3b9ad78cb7d7f4b37b675c6bc0cbc8d84c9fa5f2ca3f)

### PR DESCRIPTION
**Pull request by StrongestNumber9**
…ation

Fixes #36

### CODE REVIEW

**Review by Laukkala — COMMENTED** [review]
Deletions look good to me. Doesn't look like anything essential was removed.

File: zeppelin-zengine/src/test/java/org/apache/zeppelin/interpreter/launcher/SparkInterpreterLauncherTest.java [hunk 1]
```diff
@@ -237,37 +232,27 @@ public void testYarnClusterMode_2() throws IOException {
...
     assertTrue(client instanceof ExecRemoteInterpreterProcess);
     ExecRemoteInterpreterProcess interpreterProcess = (ExecRemoteInterpreterProcess) client;
     assertEquals("spark", interpreterProcess.getInterpreterSettingName());
     assertTrue(interpreterProcess.getInterpreterDir().endsWith("/interpreter/spark"));
-    assertTrue(interpreterProcess.getLocalRepoDir().endsWith("/local-repo/groupId"));
     assertEquals(zConf.getInterpreterRemoteRunnerPath(), interpreterProcess.getInterpreterRunner());
     assertTrue(interpreterProcess.getEnv().size() >= 3);
     assertEquals(sparkHome, interpreterProcess.getEnv().get("SPARK_HOME"));
     assertEquals("true", interpreterProcess.getEnv().get("ZEPPELIN_SPARK_YARN_CLUSTER"));
-    String sparkJars = "jar_1," +
```
Comment (kortemik) [inline zeppelin-zengine/src/test/java/org/apache/zeppelin/interpreter/launcher/SparkInterpreterLauncherTest.java:255]: why these are removed?

**Review by kortemik — CHANGES_REQUESTED** [review]

File: zeppelin-zengine/src/test/java/org/apache/zeppelin/interpreter/launcher/SparkInterpreterLauncherTest.java (hunk 1 above)
Comment (StrongestNumber9) [reply zeppelin-zengine/src/test/java/org/apache/zeppelin/interpreter/launcher/SparkInterpreterLauncherTest.java:255]: Seems like I have removed too much and should hav removed only the local repo part of it

Comment (StrongestNumber9) [reply zeppelin-zengine/src/test/java/org/apache/zeppelin/interpreter/launcher/SparkInterpreterLauncherTest.java:255]: Seems like I have also disabled this test due to

`@Ignore("This test is broken: java.lang.RuntimeException: No such note: note1")`

Was wondering why it hadn't broken the test, most likely related to issue #34

Comment (StrongestNumber9) [reply zeppelin-zengine/src/test/java/org/apache/zeppelin/interpreter/launcher/SparkInterpreterLauncherTest.java:255]: That is now readded by commit f616b3f74d597a50c66cd205a6a0d02bb4856bd5

**Review by kortemik — APPROVED** [review]

**Pull request merged**